pkg/apis/kubeone/helpers.go

```diff
@@ -119,10 +119,6 @@ func (h *HostConfig) SetLeader(leader bool) {
 	h.IsLeader = leader
 }
 
-func (c KubeOneCluster) OperatingSystemManagerEnabled() bool {
-	return c.OperatingSystemManager != nil && c.OperatingSystemManager.Deploy
-}
-
```

Comment:
I still consider this as a safer option since we won't have to depend on the defaulting for this field in code to avoid nil pointer exceptions.

Reply:
I totally agree, it's safer from "avoid NPE at any costs" perspective. Sometime having an NPE is a good thing though. NPE is a signal that we borked something, and if we suppress it with safety, we'd never know until it's too late. I'd prefer we never had `OperatingSystemManagerEnabled()` in the first place. In that case, probably, we'd hit the bug much earlier and had the fix in the v1.5.0 release :shrug:. But what's done is done.

Now let's see what NPEs have to say to us :joy:.